instrumentation/opentelemetry-api-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/opentelemetryapi/metrics/ApplicationDoubleSumObserver.java

```diff
@@ -76,6 +76,14 @@ protected Builder(io.opentelemetry.api.metrics.DoubleSumObserver.Builder agentBu
       return this;
     }
 
+    @Override
+    public DoubleSumObserver.Builder setCallback(Callback<DoubleResult> callback) {
+      agentBuilder.setCallback(
```

Comment:
@jkwatson No clue what I'm doing here - running a callback with a lambda that itself runs a function seems really funky. I want my micrometer! 😿